util/chplenv/chpl_gmp.py

```diff
@@ -24,7 +24,7 @@ def get():
 
         if os.path.exists(os.path.join(gmp_subdir, 'include', 'gmp.h')):
             gmp_val = 'gmp'
-        elif target_platform.startswith('cray-x'):
+        elif target_platform.startswith('cray-x') or target_platform.startswith('cross-'):
```

Comment:
It actually does pass the GMP tests.  I've left the default as system but moved the test away from the cray-x stuff.